modules/machineset-yaml-azure.adoc

```diff
@@ -73,17 +112,37 @@ spec:
           sshPublicKey: ""
           subnet: <infrastructureID>-<role>-subnet <1> <2>
```

Comment:
the subnet value does not seem to be a fixed value. Given infrastructureID is qeci-22538-spdj7, in some clusters it is like qeci-22538-worker-subnet, in some clusters it is like qeci-22538-spdj7-worker-subnet, we can add a comment, you can obtain the subnet by running the following command: 
```
  $  oc -n openshift-machine-api \
    -o jsonpath='{.spec.template.spec.providerSpec.value.subnet}{"\n"}' \
    get machineset/<infrastructureID>-worker-centralus1
```